doc-content/enterprise-only/openshift/secrets-central-create-proc.adoc

```diff
@@ -5,7 +5,16 @@ If you are planning to deploy {CENTRAL}
 ifdef::PAM[]
 or {CENTRAL} Monitoring
 endif::PAM[]
-in your OpenShift environment, you must create an SSL certificate for {CENTRAL} and provide it to your OpenShift environment as a secret. Do not use the same certificate and keystore for {CENTRAL} and for {KIE_SERVER}.
+in your OpenShift environment, 
+ifeval::["{context}"!="openshift-ansible-playbook"]
+you must create an SSL certificate for {CENTRAL} and provide it to your OpenShift environment as a secret.
+endif::[]
+ifeval::["{context}"=="openshift-ansible-playbook"]
+this component uses an SSL certificate to provide HTTPS access. The deployment can create the secret automatically. However, in production environments an automatically created secret can cause access issues. To avoid these issues, create an SSL certificate for {CENTRAL} and provide it to your OpenShift environment as a secret.
```

Comment:
Automatically created secrets should not be used in production environment. It is same as for example secrets (used for templates).